ngraph/core/src/op/and.cpp

```diff
@@ -61,12 +63,6 @@ namespace logand
         switch (arg0->get_element_type())
         {
             NGRAPH_TYPE_CASE(evaluate_logand, boolean, arg0, arg1, out, broadcast_spec);
-            NGRAPH_TYPE_CASE(evaluate_logand, i32, arg0, arg1, out, broadcast_spec);
-            NGRAPH_TYPE_CASE(evaluate_logand, i64, arg0, arg1, out, broadcast_spec);
-            NGRAPH_TYPE_CASE(evaluate_logand, u32, arg0, arg1, out, broadcast_spec);
-            NGRAPH_TYPE_CASE(evaluate_logand, u64, arg0, arg1, out, broadcast_spec);
-            NGRAPH_TYPE_CASE(evaluate_logand, f16, arg0, arg1, out, broadcast_spec);
-            NGRAPH_TYPE_CASE(evaluate_logand, f32, arg0, arg1, out, broadcast_spec);
```

Comment:
Shouldn't you update `has_evaluate()` types too?

https://github.com/openvinotoolkit/openvino/blob/37ff06e5273826d3bf65902ae69494caf6d76f06/ngraph/core/src/op/and.cpp#L80-L95

Reply:
Good point. Removed